easybuild/toolchains/fft/intelfftw.py

```diff
@@ -90,6 +90,13 @@ def _set_fftw_variables(self):
         # filter out libraries from list of FFTW libraries to check for if they are not provided by Intel MKL
         check_fftw_libs = [lib for lib in fftw_libs if lib not in ['dl', 'gfortran']]
         fftw_lib_exists = lambda x: any([os.path.exists(os.path.join(d, "lib%s.a" % x)) for d in fft_lib_dirs])
+        if not fftw_lib_exists(check_fftw_libs[0]) and LooseVersion(imklver) >= LooseVersion("10.2"):
+            # MKL >= 10.2 include fftw3xc and fftw3xf interfaces in LIBBLAS=libmkl_gf/libmkl_intel
+            # See https://software.intel.com/en-us/articles/intel-mkl-main-libraries-contain-fftw3-interfaces
+            check_fftw_libs.pop(0)
+            # and reduce functionality with MPI if wrappers are not built
+            if check_fftw_libs[0].startswith("fftw3x_cdft"):
+                check_fftw_libs.pop(0)
```

Comment:
Hmm, this is very dependent on the order in `check_fftw_libs`...
Can we make things more direct, by comparing directly with what should be removed from `check_fftw_libs` rather than making assumptions on order?

It's also not very clear to me what's going on here, can you clarify @bartoldeman?